### DISCUSSION TITLE
Internet provider

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#592] [General]
We recently moved to a new house and would like to know what internet provider do you guys use?
What are the pros/cons?
Would like to hear different opinions and check what companies work in our area.

**Comment by hernessteve** [comment]
We are with the sectel us.
No complaints.

**Comment by MirkosMangle** [comment]
Did you hear about h2o-wireless?
We are their customers for years now and would like to change.
[https://h2o-wireless.pissedconsumer.com/customer-service.html](https://h2o-wireless.pissedconsumer.com/customer-service.html) that's the way to contact their customer service in case you have questions about their services.
